lua/core/menu/select.lua

```diff
@@ -70,16 +111,31 @@ m.redraw = function()
       if (i > 2 - m.pos) and (i < m.len - m.pos + 3) then
         screen.move(0,10*i)
         local line = m.list[i+m.pos-2].name
+        local is_fave = "   "
         if(i==3) then
           screen.level(15)
+          m.selected_script = m.list[i+m.pos-2]
         else
           screen.level(4)
         end
-        screen.text(string.upper(line))
+        if contains(m.favorites, m.list[i+m.pos-2]) then is_fave = "* " else is_fave = "  " end
+        screen.text(is_fave .. string.upper(line))
       end
     end
   end
   screen.update()
 end
 
+local function noop(_)
+end
+
+m.add_favorite = function()
```

Comment:
both of these are kinda mega hacks -- if it'd be preferable to do this is a lua native way i'm totally happy to go that route.

Reply:
I haven’t looked at the code in detail but at first glance I’d consider maintaining favorites in a table and just using `tab.save` and `tab.load` functions in the `tabutil` module.

https://github.com/monome/norns/blob/main/lua/lib/tabutil.lua#L93

Reply:
ah yeah, that's way better. i'll get that pushed up in a few